k8s/aat/common/family-public-law/rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: admin-fpl
+  namespace: family-public-law
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: admin
+subjects:
+  - kind: Group
+    name: system:authenticated
```

Comment:
this shouldn't be authenticated it should be the user group

Reply:
Yes, but i guess it needs to be reference by objectId?

Reply:
Yes I know, do we have way of dynamically extracting it?

Reply:
we look it up in the pipeline and put it in, you would likely need to hardcode it here